spec/openapi.yaml

```diff
@@ -6468,6 +6468,84 @@ components:
             The subject field is used to denote subject of the MMS message and
             has a maximum size of 64 characters long
           example: New Products
+    DetailMessage:
+      title: Message
+      required:
+        - content
+        - destination_number
+      type: object
+      properties:
+        content:
+          maxLength: 5000
+          minLength: 1
+          type: string
+          description: Content of the message
+          example: Hello world!
+        destination_address:
+          maxLength: 15
+          minLength: 1
+          type: string
+          description: Destination number of the message
+          example: '+61491570156'
```

Comment:
See suggestions for phone numbers below. You could use a shortcode to for one side of the conversation like `8787` or `7500`.